design-system/materials/resets.css

```diff
@@ -0,0 +1,41 @@
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
```

Comment:
I tracked the currently failing Percy test related to `CollapsiblePanel` component down to this new rules. I makes the icon a little bigger and it also moves the title to the right.

Perhaps this is intended since we were actually using it like this in the applications (through `app-kit`) so maybe we can approve those changes in Percy.

Reply:
Yeah I think we can approve that.